libraries/AP_SpdHgtControl/AP_SpdHgtControl.h

```diff
@@ -10,11 +10,10 @@
   own class.
  */
 
-#include <AP_Common/AP_Common.h>
```

Comment:
I couldn't convince myself this was safe anywhere (everything still compiled, but seemed dangerous). You are also now transatively getting this via `AP_Vehicle.h` probably which would hide anything.

Reply:
What do you think could go wrong without this header not being included?  I'm not seeing anything in there setting things that other places will `#ifdef` or similar - I think you'd simply get compilation failures.

We get identical binaries before and after this PR (on fmuv2), so as per devcall I'm merging.

Thanks!